cmd/register/register_util.go

```diff
@@ -275,34 +275,76 @@ func hydrateTaskSpec(task *admin.TaskSpec, sourceCode, sourceUploadPath, version
 	return nil
 }
 
-func validateLaunchSpec(lpSpec *admin.LaunchPlanSpec) error {
-	if lpSpec == nil {
+func validateLPWithSchedule(lpSpec *admin.LaunchPlanSpec) error {
+	if lpSpec.EntityMetadata == nil || lpSpec.EntityMetadata.Schedule == nil {
```

Comment:
do you still want to keep the check for if the spec itself is nil before dereferencing?

Reply:
ah nvm I see it's been moved below.